querydsl-spatial/src/main/java/com/querydsl/spatial/path/GeometryPath.java

```diff
@@ -139,7 +139,7 @@ public GeometryPath(String var) {
         return polygon;
     }
 
-    public PolyhedralSurfacePath<PolyHedralSurface> asPolygHedralSurface() {
+    public PolyhedralSurfacePath<PolyHedralSurface> asPolyHedralSurface() {
```

Comment:
I just noticed that Geolatte also uses PolyHedral so it might be best to leave it.